calico_containers/pycalico/datastore.py

```diff
@@ -326,11 +326,13 @@ def get_host_bgp_ips(self, hostname):
             return (ipv4, ipv6)
 
     @handle_errors
-    def get_ip_pools(self, version):
+    def get_ip_pools(self, version, ipam_only=False):
```

Comment:
Yeah I will change that!  It was originally because I wanted a lack of value in the JSON to be "False", but that really isn't necessary.